### DISCUSSION TITLE
EvilCrow RF programmable buttons

### DISCUSSION THREAD

**Discussion by crocojohn** [rfquack/RFQuack#14] [General]
Hello!
Received EvilCrow RF from BLE beacon store. But 1 button was broken :(
It is 1st progammable button in a row of three buttons. Another buttons works.
Can you help me how to repair it? Or any advises how to do with this case? ((

**Comment by phretor** [comment]
Hello. Mh, this sounds like a hardware-related question, also not regarding RFQuack.

**Reply by crocojohn** [reply]
thank you!
